.chloggen/datadog-exporter-span-links-support.yaml

```
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: datadogexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Adds support for OpenTelemetry span links when converting OTLP spans (span links are added as metadata to the converted span).
```

I think here we don't need the context about this being about OpenTelemetry (as opposed to in the Datadog Agent where it was necessary)
```suggestion
note: Adds support for span links (span links are added as metadata to the converted span).
```